src/CampaignsAdmin/CampaignAdmin.tsx

```diff
@@ -4,14 +4,25 @@ import { CampaignList } from './CampaignList';
 import { NewCampaignForm } from './NewCampaignForm';
 import { PlusOutlined, CloseOutlined } from '@ant-design/icons';
 import 'styles/pages/campaignAdmin.scss';
-
+import { useHistory } from 'react-router';
+import { useAuth } from '../contexts/Auth';
 function CampaignAdmin() {
   const [drawerVisibility, setDrawerVisibility] = useState(false);
 
   const toggleDrawer = () => {
     setDrawerVisibility(!drawerVisibility);
   };
 
+  const { user, signOut } = useAuth();
```

Comment:
This feels like a smell. The Campaign Admin shouldn't have to know anything about signing out. If you need to display a button in multiple places it's better to abstract unto its own Component and/or leave it on the header for the whole project